utils/udfsdk/CMakeLists.txt

```diff
@@ -11,9 +11,11 @@ set(udfsdk_LIB_SRCS
 )
 
 columnstore_library(udfsdk ${udfsdk_LIB_SRCS})
-columnstore_link(udfsdk PRIVATE loggingcpp)
+columnstore_link(udfsdk PRIVATE loggingcpp messageqcpp)
```

Comment:
Are you sure you need to add messageqcpp here?

Reply:
at this moment we need ByteStream here